app/src/main/java/com/github/se/signify/MainActivity.kt

```diff
@@ -118,11 +118,6 @@ fun SignifyAppPreview() {
             profilePictureUrl = null, // Replace with actual URL or null
```

Comment:
This is way too many arguments for a screen. I think it should be passed as a view model or some class.